init-tools.ps1

```diff
@@ -57,6 +57,61 @@ if (!(Test-Path "$BUILD_TOOLS_PACKAGE_PATH\init-tools.cmd"))
     exit 1
 }
 
+# Bring down the CLI for build tools
+$DOTNET_PATH=$BUILD_TOOLS_PATH + "\dotnetcli"
+
+Write-Host "Installing Build Tools CLI version..."
+if (!(Test-Path "$DOTNET_PATH"))
+{
+     mkdir "$DOTNET_PATH"
+}
+
+$DOTNET_VERSION = Get-Content "$RepoRoot\BuildToolsCLIVersion.txt"
+$DOTNET_ZIP_NAME="dotnet-dev-win-x64." + $DOTNET_VERSION + ".zip"
```

Comment:
@brthor If I'm downloading a script just to download another file, I might as well just stick with the code to download preview2 bits directly and skip jumping through hoops :). If the current obtain script doesn't allow for preview2 bits to be downloaded then the simplest approach is just to keep the code as-is :)